src/marker.rs

```diff
@@ -619,6 +619,13 @@ impl<'py> Python<'py> {
         unsafe { PyObject::from_borrowed_ptr(self, ffi::Py_None()) }
     }
 
+    /// Gets the Python builtin value `Ellipsis`, or `...`.
+    #[allow(non_snake_case)] // the Python keyword starts with uppercase
+    #[inline]
+    pub fn Ellipsis(self) -> PyObject {
```

Comment:
Rather than returning `PyObject` here it would probably be nicer if we returned `&PyEllipsis`, and created a new struct `types::PyEllipsis`. (Would avoid the need to increase a reference count here.)

That would mean that `x.is_instance::<PyEllipsis>()` would just work instead of needing `is_ellipsis`, which does have the upside of avoiding a special method. Not necessarily saying that it's better, perhaps `is_ellipsis` is still warranted.

Also, adding `types::PyEllipsis` would pave the way for instead just having `PyEllipsis::get(py)` to retrieve the singleton, rather than adding this to `Python` which I don't think really needs this method.

Reply:
Then we should do this for None and NotImplemented too. The current pr mirrors their api.

Reply:
Agreed, and I've been wanting to do that for a while. I guess we can't chance those for 0.18.x, so perhaps we merge this as-is and I'll probably raise a follow-up PR to adjust all three on the way to 0.19?

Reply:
👍